test/ufuzz.js

```diff
@@ -11,115 +15,219 @@
+    '({})', // wrapped the object causes too many syntax errors in statements
+    '"foo"',
+    '"bar"' ];
 
 var BINARY_OPS_NO_COMMA = [
-  ' + ', // spaces needed to disambiguate with ++ cases (could otherwise cause syntax errors)
-  ' - ',
-  '/',
-  '*',
-  '&',
-  '|',
-  '^',
-  '<<',
-  '>>',
-  '>>>',
-  '%',
-  '&&',
-  '||',
-  '^' ];
+    ' + ', // spaces needed to disambiguate with ++ cases (could otherwise cause syntax errors)
+    ' - ',
+    '/',
+    '*',
+    '&',
+    '|',
+    '^',
+    '<<',
+    '>>',
+    '>>>',
+    '%',
+    '&&',
+    '||',
+    '^' ];
 
 var BINARY_OPS = [','].concat(BINARY_OPS_NO_COMMA);
 
 var ASSIGNMENTS = [
-  '=',
-  '=',
-  '=',
-  '=',
-  '=',
-  '=',
-
-  '==',
-  '!=',
-  '===',
-  '!==',
-  '+=',
-  '-=',
-  '*=',
-  '/=',
-  '&=',
-  '|=',
-  '^=',
-  '<<=',
-  '>>=',
-  '>>>=',
-  '%=' ];
+    '=',
+    '=',
+    '=',
+    '=',
+    '=',
+    '=',
+
+    '==',
+    '!=',
+    '===',
+    '!==',
+    '+=',
+    '-=',
+    '*=',
+    '/=',
+    '&=',
+    '|=',
+    '^=',
+    '<<=',
+    '>>=',
+    '>>>=',
+    '%=' ];
 
 var UNARY_OPS = [
-  '--',
-  '++',
-  '~',
-  '!',
-  'void ',
-  'delete ', // should be safe, even `delete foo` and `delete f()` shouldn't crash
-  ' - ',
-  ' + ' ];
+    '--',
+    '++',
+    '~',
+    '!',
+    'void ',
+    'delete ', // should be safe, even `delete foo` and `delete f()` shouldn't crash
+    ' - ',
+    ' + ' ];
 
 var NO_COMMA = true;
 var MAYBE = true;
-var NESTED = true;
 var CAN_THROW = true;
 var CANNOT_THROW = false;
 var CAN_BREAK = true;
+var CANNOT_BREAK = false;
 var CAN_CONTINUE = true;
+var CANNOT_CONTINUE = false;
+var CANNOT_RETURN = true;
+var NOT_GLOBAL = true;
+var IN_GLOBAL = true;
+var ANY_TYPE = false;
+var NO_DECL = true;
 
 var VAR_NAMES = [
-  'foo',
-  'bar',
-  'a',
-  'b',
-  'undefined', // fun!
-  'eval', // mmmm, ok, also fun!
-  'NaN', // mmmm, ok, also fun!
-  'Infinity', // the fun never ends!
-  'arguments', // this one is just creepy
-  'Math', // since Math is assumed to be a non-constructor/function it may trip certain cases
-  'let' ]; // maybe omit this, it's more a parser problem than minifier
+    'foo',
+    'bar',
+    'a',
+    'b',
+    'c', // prevent redeclaring this, avoid assigning to this
+    'undefined', // fun!
+    'eval', // mmmm, ok, also fun!
+    'NaN', // mmmm, ok, also fun!
+    'Infinity', // the fun never ends!
+    'arguments', // this one is just creepy
+    'Math', // since Math is assumed to be a non-constructor/function it may trip certain cases
+    'let' ]; // maybe omit this, it's more a parser problem than minifier
 
 var TYPEOF_OUTCOMES = [
-  'undefined',
-  'string',
-  'number',
-  'object',
-  'boolean',
-  'special',
-  'unknown',
-  'symbol',
-  'crap' ];
+    'undefined',
+    'string',
+    'number',
+    'object',
+    'boolean',
+    'special',
+    'unknown',
+    'symbol',
+    'crap' ];
 
 var FUNC_TOSTRING = [
     "Function.prototype.toString = function() {",
```

Comment:
Ah yeah I missed this. Github should put these comments somewhere chronologically as well..